docs/metrics.md

```diff
@@ -0,0 +1,195 @@
+    * Click "OK"
+  * Click to delete a single container
+    * Click "OK"
+  * Click to add a container
+    * Click "OK"
+* Do people sort the tabs?
+  * Click sort
+    * \+ Number of tabs when clicked
+  * Average number of container tabs when sort was clicked
+* Do users show and hide container tabs?
+  * Click hide
+    * \+ Number of tabs when clicked
+    * \+ Number of hidden containers when clicked
+  * Click show
+    * \+ Number of tabs when clicked
+    * \+ Number of shown containers when clicked
+* Do users move container tabs to new windows?
+  * Click move
+    * \+ Number of tabs when clicked
+  * Average number of container tabs when new window was clicked
+* How many containers do users have hidden at the same time? (when should we measure this? each time a container is hidden?)
+* Do users pin container tabs? (do we have existing Telemetry for pinning?)
+* Do users change URLs in a container tab? (sounds like it could be a flood unless we only record the first URL change?)
+
+### Follow-up Questions
+
+What are some follow-up questions we anticipate we will ask based on any of the
+above answers/data?
+
+* What is the average lifespan of a container tab? Is that longer or shorter than a regular tab? (if we don't have data on the latter, the former probably isn't worth gathering data on since we will have nothing to compare it to).
+
+## Data Collection
+
+### Server Side
+There is currently no server side component to Containers.
+
+### Client Side
+Containers will use Test Pilot Telemetry with no batching of data.  Details
+of when pings are sent are below, along with examples of the `payload` portion
+of a `testpilottest` telemetry ping for each scenario.
+
+* The user clicks on a container name to open a tab in that container
+
+```js
+  {
+    "uuid": <uuid>,
+    "userContextId": <userContextId>,
+    "clickedContainerTabCount": <number-of-tabs-in-the-container>,
+    "event": "open-tab",
+    "eventSource": ["tab-bar"|"pop-up"|"file-menu"]
+  }
+```
+
+* The user clicks "Edit Containers" in the pop-up
+
+```js
+  {
+    "uuid": <uuid>,
+    "event": "edit-containers"
+  }
+```
+
+* The user clicks OK after clicking on a container edit icon in the pop-up
+
+```js
+  {
+    "uuid": <uuid>,
+    "userContextId": <userContextId>,
+    "event": "edit-container"
+  }
+```
+
+* The user clicks OK after clicking on a container delete icon in the pop-up
+
+```js
+  {
+    "uuid": <uuid>,
+    "userContextId": <userContextId>,
+    "event": "delete-container"
+  }
+```
+
+* The user clicks OK after clicking to add a container in the pop-up
+
+```js
+  {
+    "uuid": <uuid>,
+    "userContextId": <userContextId>,
+    "event": "add-container"
+  }
+```
+
+* The user clicks the sort button/icon in the pop-up
+
+```js
+  {
+    "uuid": <uuid>,
+    "event": "sort-tabs",
+    "totalContainerTabsCount": <number-of-all-container-tabs>
+    "totalNonContainerTabsCount": <number-of-all-non-container-tabs>
+  }
```

Comment:
Add the shownContainersCount here, so we have an idea of how many containers they are sorting through.